config/routes.rb

```diff
@@ -146,6 +149,10 @@
         end
       end
 
+      resources :lectures, concerns: :paginatable do
+        get :access_link, on: :member
```

Comment:
It doesn't work. W/O `on: :member` `load_and_authorize_resource` doesn't work correctly in lectures_controller (`@lecture` doesn't gets loaded)